docs/dev/building.md

```diff
@@ -77,20 +84,36 @@ If you don't specify a build mode, the `build.ninja` will contain configuration
 
 [Ninja]: https://ninja.org/
 
+In the CMake-based building system, the build modes are mapped to the configuration names:
+
+|  mode      | config           |
+| ---------- | ---------------- |
+| `dev`      | `Dev`            |
+| `release`  | `RelWithDebInfo` |
+| `debug`    | `Debug`          |
+| `sanitize` | `Sanitize`       |
+| `coverage` | `Coverage`       |
+
 ## Building
 
 To build Scylla executables and tests, run:
 
 ```console
-./tools/toolchain/dbuild ninja build
+./tools/toolchain/dbuild ninja -C build
 ```
 
 This will build Scylla for the build mode specified by the `configure.py` command.
```

Comment:
Please audit all the places where the words "build mode" or "config" are used, and change all of them to use the same terminology. You can mention in one place also the older or alternative terminology - but it's not good that you randomly switch between the two terminologies.

Reply:
please note, i am not using these two terminologies interchangeably, or "randomly" switching between them. "mode" is used along with `configure.py`, while `config` is used along with CMake and ninja.

Reply:
ahh, i did find some places where i was using "mode" by mistake. corrected.